internal/servers/controller/handlers/outgoing_interceptor.go

```diff
@@ -34,6 +35,15 @@ func OutgoingInterceptor(ctx context.Context, w http.ResponseWriter, m proto.Mes
 			return nil
 		}
 		fields := m.GetAttributes().GetFields()
+		if m.GetCommand() == "token" {
```

Comment:
perhaps define a const for "token" which is used a few times.

Reply:
It's used a few times but in different ways. In one case it's a command, in the other it's a field name. So it could be e.g. `tokenString` but `tokenField` and `tokenCommand` don't fit...which makes me wonder if there is then value since it's purely to get rid of a string and not to convey useful info.